pages/docs/tutorials/jvm-get-started.md

```diff
@@ -2,61 +2,81 @@
 type: tutorial
 layout: tutorial
 title:  "Getting Started with IntelliJ IDEA"
-description: "This tutorial walks you through creating a simple Hello World application using IntelliJ IDEA."
-authors: Hadi Hariri, Roman Belov
-date: 2020-01-23
+description: "This tutorial demonstrates how to use IntelliJ IDEA for creating a console application. "
+authors: Kate Volodko
+date: 2020-07-07
 showAuthorInfo: false
 ---
-## Setting up the environment
 
-In this tutorial we're going to use IntelliJ IDEA. To get started, install a recent version of IntelliJ IDEA.
+To get started, install a recent version of [IntelliJ IDEA](http://www.jetbrains.com/idea/download/index.html).
 
-Kotlin is bundled with IntelliJ IDEA starting from version 15.
+## Create an application 
 
-You can download the free [Community Edition][intellijdownload] (or full-fledged [Ultimate Edition][intellijdownload]) from the [JetBrains website][jetbrains].
+Once you've installed IntelliJ IDEA, it's time to create your first Kotlin application.
 
-As an alternative to using IntelliJ IDEA, you can compile and execute Kotlin applications using the command line compiler. For details, see [Working with the Command Line Compiler][getting_started_command_line].
+1. In IntelliJ IDEA, select **File** \| **New** \| **Project**.
+2. In the panel on the left, select **Kotlin**.
+3. Enter a project name, select **Console Application** as the project template, and click **Next**.
+   
+   ![Create a console application]({{ url_for('tutorial_img', filename='getting-started/jvm-new-project-1.png') }})
+   
+   By default, your project will use the build system Gradle with Kotlin DSL.
```

Comment:
Maybe add a link to https://docs.gradle.org/current/userguide/kotlin_dsl.html?
It seems to me that a reader of "how to create hello world in Kotlin " isn't likely to know about the Kotlin Gradle DSL.